File: blocksandstuff-blocks/src/main/java/org/everbuild/blocksandstuff/blocks/group/placement/IntoPlacementRule.java
package org.everbuild.blocksandstuff.blocks.group.placement;

import net.minestom.server.instance.block.Block;
import net.minestom.server.instance.block.rule.BlockPlacementRule;

public interface IntoPlacementRule {
    BlockPlacementRule createRule(Block block);
}

File: blocksandstuff-blocks/src/main/kotlin/org/everbuild/blocksandstuff/blocks/group/placement/IntoPlacementRule.kt
package org.everbuild.blocksandstuff.blocks.group.placement

import net.minestom.server.instance.block.Block
import net.minestom.server.instance.block.rule.BlockPlacementRule

interface IntoPlacementRule {
    fun createRule(block: Block): BlockPlacementRule
}
